subsys/mgmt/mcumgr/Kconfig

```diff
@@ -164,6 +164,15 @@ config IMG_MGMT_DUMMY_HDR
 	help
 	  Returns dummy image header data for imgr functions,
 	  useful when there are no images present, Eg: unit tests.
+
+config IMG_MGMT_REJECT_DIRECT_XIP_MISMATCHED_SLOT
+	bool "Reject Direct-XIP applications with mismatched address"
+	help
+	  When enabled, the mcumgr will compare base address of application,
+	  encoded into .bin file header with use of imgtool, on upload and will
```

Comment:
.bin -> <i>image header/signed image header</i>
The base address can be included to the image header using  imgtool with --rom_fixed option.

Reply:
@de-nordic ^^